src/t/d10.clj

```diff
@@ -0,0 +1,85 @@
+(ns t.d10
+  (:gen-class)
+  (:require
+    [clojure.string :as cljstr]
+    [clojure.set :as cljset]
+    [clojure.core.match :refer [match]]))
+
+(def sample
+"89010123
+78121874
+87430965
+96549874
+45678903
+32019012
+01329801
+10456732")
+
+(defn parse-input
+  [input]
+  (->> (cljstr/split-lines input)
+       (map vec)
```

Comment:
I don't think this line does anything. It turns the per-line strings into vectors of characters, but you then `map-indexed` over them anyway. `(map-indexed f str)` will do the same as `(map-indexed f (vec str))`.

Reply:
yes, at the start I thought I was going to just keep a vector of vector of the characters and build from there, and then I decided I wanted another structure >.>
forgot to clean-up again